### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by zilton7**
In this milestone:
- Prepared the directory structure
- Implemented the Book component
- Define the Redux actions and reducer
- Initialize the Redux store

Thanks for the review.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </React.StrictMode>,
   document.getElementById('root'),
 );
```
Comment (arslanbisharat) [inline src/index.js]: - [x]  index.js should have example books to initial state object. 
InitialBooks

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BooksList />
+      <BooksForm />
     </div>
   );
 }
```
Comment (arslanbisharat) [inline src/App.js]: - [x] App.js should be placed in `components` folder.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @taiwo2 and @zilton7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Best, 
Arslan Bisharat
@arslanbisharat

File: src/index.js (hunk 1 above)
Comment (zilton7) [reply src/index.js]: We have done this inside reducers/books.js :)

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @taiwo2  & @zilton7**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**